### DISCUSSION TITLE
Dropdown

### DISCUSSION THREAD

**Discussion by 5675sp** [h2oai/wave#944] [General]
Is there a way to know when the value of a dropdown is change without setting `Trigger` to `True`(I just want to listen to the change in the dropdown and not submitted the entire form)? 

I know that I can listen to the change by using `q.args....`, but `q.args....` doesn't work unless the server function is call

**Comment by mturoci** [comment]
Hi @5675sp, not sure I fully understand your question. Would you like to listen for changes on client (javascript) not server (python) or do you want `q.args` to have submitted only the dropdown value without the rest of the form values?

**Reply by 5675sp** [reply]
Hello @mturoci, I want `q.args` to have submitted only the dropdown value without the rest of the form values. I just want to know the new value of the dropdown when it changes. But, it seems that it's a current problem that is been worked on.

**Reply by lo5** [reply]
This is not possible. All the form fields are submitted unconditionally.
